imls-backend/README.md

```diff
@@ -1,127 +1,116 @@
-# Backend
-
-## Developer configuration
-
-This is for local configuration only and should never be run in production.
-
-### Run Postgrest
-
-The backend stack relies on a postgres image that includes pgcrypto and pgjwt. This must be built before running.
-
-- Run `docker build -t imls:postgres -f Dockerfile.pgjwt .`
-- Run `docker-compose up`
-
-### Run Migrations
-
-- install [dbmate](https://github.com/amacneil/dbmate)
-- configure your `.env` for dbmate:
-  - `DATABASE_URL="postgres://postgres:imlsimls@localhost:5432/imls?sslmode=disable"`
-- Open a separate terminal and run `dbmate up`
-
-## Connect to the DB via CLI
-
-- `psql -h localhost -U postgres -W`
-- Enter the password from the docker compose file
-- Connect to the database: `\c imls`
-- View schemas `\dn`
-- View users `\du`
-- Use a schema `SET schema 'imlswifi';`
-- List tables `\dt`
-- List views `\dv`
-
-## Connect to the DB via pgAdmin
-
-- Ensure you've started the Docker containers as outlined above
-- Open a browser and go to `localhost:8080`
-- Enter the pgAdmin creds from docker-compose
-  - imls@gsa.gov
-  - imlsimls
-- Click 'Add New Server'
-- In the 'General' tab
-  - Choose a name (can be anything)
-- In the 'Connection' tab
-  - The 'Host name/address' is `db`
-  - The 'Port' is `5432`
-  - The 'Username' is `postgres`
-  - The 'Password' is `imlsimls`
-
-## Query the DB
-
-- `curl -s http://127.0.0.1:3000/presences`
-- Fields from the `presences` table
-  - presence_id SERIAL PRIMARY KEY,
-  - start_time TIMESTAMPTZ NOT NULL,
-  - end_time TIMESTAMPTZ NOT NULL,
-  - fscs_id VARCHAR(16) NOT NULL [FOREIGN KEY],
-  - sensor_id SERIAL [FOREIGN KEY],
-  - manufacturer_index INTEGER,
-
-## Query Stored Procedures and Functions from the DB
```

Comment:
I think the sections about querying the local DB might still be necessary for when @anagradova and @sknep work together. Or perhaps moved to another location.

Reply:
I pulled the DB info into a new, separate file called DATABASE_NOTES.md and linked to it in the README.